diff --git a/packages/sysutils/busybox/scripts/init b/packages/sysutils/busybox/scripts/init
index 19afb896..023f9e5b 100755
--- a/packages/sysutils/busybox/scripts/init
+++ b/packages/sysutils/busybox/scripts/init
@@ -1098,7 +1098,7 @@ prepare_sysroot() {
   mount --move /flash /sysroot/flash
   mount --move /storage /sysroot/storage
 
-  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/$(uname -r)/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
+  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/4.19.172/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
     echo ""
     echo "NEVER TOUCH boot= in syslinux.conf / cmdline.txt!"
     echo "If you don't know what you are doing,"
diff --git a/packages/sysutils/systemd/scripts/kernel-overlays-setup b/packages/sysutils/systemd/scripts/kernel-overlays-setup
index 67c5ca26..b7fcdc8b 100755
--- a/packages/sysutils/systemd/scripts/kernel-overlays-setup
+++ b/packages/sysutils/systemd/scripts/kernel-overlays-setup
@@ -5,7 +5,7 @@
 
 SYSTEM_OVERLAYS_DIR=/usr/lib/kernel-overlays
 OVERLAY_CONFIG_DIR=/storage/.cache/kernel-overlays
-KVER=$(uname -r)
+KVER=4.19.172
 MODULES_DIR="/var/lib/modules/${KVER}"
 FIRMWARE_DIR="/var/lib/firmware"
 USER_FIRMWARE_DIR="/storage/.config/firmware"
