diff --git a/packages/jelos/sources/scripts/batocera-config b/packages/jelos/sources/scripts/batocera-config
index 8948fb6f..c13ae7b7 100755
--- a/packages/jelos/sources/scripts/batocera-config
+++ b/packages/jelos/sources/scripts/batocera-config
@@ -244,11 +244,11 @@ if [ "$command" == "module" ];then
 	modulename="$extra1"
 	map="$extra2"
 	# remove in all cases
-	rmmod /lib/modules/`uname -r`/extra/${modulename}.ko >> $log
+	rmmod /lib/modules/4.19.172/extra/${modulename}.ko >> $log
 
         if [ "$mode" == "load" ];then
 	        echo "loading module $modulename args = $map" >> $log
-		insmod /lib/modules/`uname -r`/extra/${modulename}.ko $map >> $log
+		insmod /lib/modules/4.19.172/extra/${modulename}.ko $map >> $log
 		[ "$?" ] || exit 1
         fi
 	exit 0
diff --git a/packages/sysutils/busybox/scripts/init b/packages/sysutils/busybox/scripts/init
index 19afb896..023f9e5b 100755
--- a/packages/sysutils/busybox/scripts/init
+++ b/packages/sysutils/busybox/scripts/init
@@ -1098,7 +1098,7 @@ prepare_sysroot() {
   mount --move /flash /sysroot/flash
   mount --move /storage /sysroot/storage
 
-  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/$(uname -r)/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
+  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/4.19.172/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
     echo ""
     echo "NEVER TOUCH boot= in syslinux.conf / cmdline.txt!"
     echo "If you don't know what you are doing,"
diff --git a/packages/sysutils/systemd/scripts/kernel-overlays-setup b/packages/sysutils/systemd/scripts/kernel-overlays-setup
index 67c5ca26..b7fcdc8b 100755
--- a/packages/sysutils/systemd/scripts/kernel-overlays-setup
+++ b/packages/sysutils/systemd/scripts/kernel-overlays-setup
@@ -5,7 +5,7 @@
 
 SYSTEM_OVERLAYS_DIR=/usr/lib/kernel-overlays
 OVERLAY_CONFIG_DIR=/storage/.cache/kernel-overlays
-KVER=$(uname -r)
+KVER=4.19.172
 MODULES_DIR="/var/lib/modules/${KVER}"
 FIRMWARE_DIR="/var/lib/firmware"
 USER_FIRMWARE_DIR="/storage/.config/firmware"
