From 626955b7bb77fb0c63d9bcfc153588831ab47ae1 Mon Sep 17 00:00:00 2001
From: johnloser-lwi <john-masterchief@hotmail.com>
Date: Wed, 14 Sep 2022 22:39:42 +0800
Subject: [PATCH] proc info

---
 packages/sysutils/busybox/scripts/init | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/sysutils/busybox/scripts/init b/packages/sysutils/busybox/scripts/init
index 19afb896..69125cf7 100755
--- a/packages/sysutils/busybox/scripts/init
+++ b/packages/sysutils/busybox/scripts/init
@@ -1100,6 +1100,9 @@ prepare_sysroot() {
 
   if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/$(uname -r)/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
     echo ""
+    ls /sysroot/usr/lib/kernel-overlays/base/lib/modules
+    uname -r
+    cat /proc/version
     echo "NEVER TOUCH boot= in syslinux.conf / cmdline.txt!"
     echo "If you don't know what you are doing,"
     echo "your installation is now broken."
-- 
2.35.1.windows.2

