diff --git a/packages/themes/thememaster/package.mk b/packages/themes/thememaster/package.mk
index 18db1b0b..dc388990 100644
--- a/packages/themes/thememaster/package.mk
+++ b/packages/themes/thememaster/package.mk
@@ -3,9 +3,9 @@
 # Copyright (C) 2022-present BrooksyTech (https://github.com/brooksytech)
 
 PKG_NAME="thememaster"
-PKG_VERSION="09bee95fea09760355777af824a7ad41dc1ae577"
+PKG_VERSION="49586ece8ffec4cf17793da91754d28d90be1542"
 PKG_ARCH="any"
-PKG_SITE="https://github.com/JohnIrvine1433/ThemeMaster"
+PKG_SITE="https://github.com/johnloser-lwi/ThemeMaster"
 PKG_URL="${PKG_SITE}.git"
 PKG_PRIORITY="optional"
 PKG_SECTION="theme"
