diff --git a/packages/sysutils/busybox/scripts/init b/packages/sysutils/busybox/scripts/init
index 69125cf7..525b8d4f 100755
--- a/packages/sysutils/busybox/scripts/init
+++ b/packages/sysutils/busybox/scripts/init
@@ -1098,7 +1098,7 @@ prepare_sysroot() {
   mount --move /flash /sysroot/flash
   mount --move /storage /sysroot/storage
 
-  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/$(uname -r)/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
+  if [ ! -d "/sysroot/usr/lib/kernel-overlays/base/lib/modules/$(echo $(uname -r) | cut -d - -f 1)/" -a -f "/sysroot/usr/lib/systemd/systemd" ]; then
     echo ""
     ls /sysroot/usr/lib/kernel-overlays/base/lib/modules
     uname -r
@@ -1124,7 +1124,7 @@ check_amlogic_dtb() {
   if grep -q "amlogic" /proc/device-tree/compatible 2>/dev/null; then
     if grep -q "official" /sysroot/etc/os-release 2>/dev/null; then
       progress "Checking Amlogic DTB"
-      if [ "$(uname -r)" = "3.14.29" ]; then
+      if [ "$(echo $(uname -r) | cut -d - -f 1)" = "3.14.29" ]; then
         DT_ID=$(cat /proc/device-tree/le-dt-id 2>/dev/null)
       else
         DT_ID=$(cat /proc/device-tree/coreelec-dt-id 2>/dev/null)
diff --git a/packages/sysutils/systemd/scripts/kernel-overlays-setup b/packages/sysutils/systemd/scripts/kernel-overlays-setup
index 67c5ca26..74498a13 100755
--- a/packages/sysutils/systemd/scripts/kernel-overlays-setup
+++ b/packages/sysutils/systemd/scripts/kernel-overlays-setup
@@ -5,7 +5,7 @@
 
 SYSTEM_OVERLAYS_DIR=/usr/lib/kernel-overlays
 OVERLAY_CONFIG_DIR=/storage/.cache/kernel-overlays
-KVER=$(uname -r)
+KVER=$(echo $(uname -r) | cut -d - -f 1)
 MODULES_DIR="/var/lib/modules/${KVER}"
 FIRMWARE_DIR="/var/lib/firmware"
 USER_FIRMWARE_DIR="/storage/.config/firmware"
